src/mui/field/ReferenceManyField.js

```diff
@@ -62,15 +68,21 @@ export class ReferenceManyField extends Component {
         }
     }
 
-    fetchReferences({ reference, record, resource, target, perPage, sort, filter } = this.props) {
+    setSort = (field) => {
```

Comment:
@djhi sorry, I made a mistake here and I need to replace these lines with:
```jsx
this.setState(
    { sort: { field, order } },
    this.fetchReferences,
);
```
How can I do this since it's merged?